cmd/nvidia-ctk/cdi/generate/generate_test.go

```diff
@@ -127,7 +124,7 @@ containerEdits:
 				vendor:        "example.com",
 				class:         "device",
 				driverRoot:    driverRoot,
-				disabledHooks: *disableHook1,
+				disabledHooks: valueOf(cli.NewStringSlice("enable-cuda-compat")),
```

Comment:
The use of arbitrary names for the variables made the intent of the tests more difficult to understand. A `valueOf` helper allows us to return the value of a string slice and list the disabled hooks in the tests.